docs/compile.dox

```diff
@@ -71,7 +71,7 @@ cmake -DCMAKE_TOOLCHAIN_FILE=CMake/x86_64-w64-mingw32.cmake .
 @endcode
 
 For more details see the article
-[CMake Cross Compiling](http://www.paraview.org/Wiki/CMake_Cross_Compiling) on
+[CMake Cross Compiling](https://www.paraview.org/Wiki/CMake_Cross_Compiling) on
```

Comment:
This page says it has moved to https://gitlab.kitware.com/cmake/community/wikis/doc/cmake/CrossCompiling so you should replace it.